libraries/networking/src/ResourceCache.cpp

```diff
@@ -16,8 +16,9 @@
 #include <cmath>
 #include <assert.h>
 
-#include <QThread>
-#include <QTimer>
+#include <QtCore/QMetaMethod>
```

Comment:
same here

Reply:
QMetaMethod as an include was removed from PacketReceiver.h.  That broke some files that included PacketReceiver.h, most likely making references to QMetaEnum, so #includes were added "downstream".